speech/unit_tests/test_client.py

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 import unittest
-
```

Comment:
This empty line "belongs" here because `unittest` is in the stdlib and `mock` is not (`unittest.mock` **is**, but only in Python 3)